htdocs/langs/en_US/other.lang

```diff
@@ -2,6 +2,7 @@
 SecurityCode=Security code
 NumberingShort=N°
 Tools=Tools
+TMenuTools=TOOLS
```

Comment:
Can you replace with a not full upcase translation "Product".
If need is to have first level top menu in uppercase, this can be addresse by an option in the css file to force case of title level